jvm/ml4ir-inference/pom.xml

```diff
@@ -10,6 +10,10 @@
   <artifactId>ml4ir-inference</artifactId>
   <packaging>jar</packaging>
 
+  <properties>
+    <jackson.version>2.7.5</jackson.version>
```

Comment:
Update this to 2.10.0

Reply:
We can re-use the jackson version from parent's pom instead of mentioning it here again. But we have to use 2.10.0 (as this version is not compatible with spring)